sip-core/src/main/java/de/ikor/sip/foundation/core/declarative/utils/DeclarativeHelper.java

```diff
@@ -24,4 +25,10 @@ public static String formatConnectorId(
       ConnectorType type, String scenarioID, String connectorId) {
     return String.format(CONNECTOR_ID_FORMAT, type.getValue(), scenarioID, connectorId);
   }
+
+  public static EndpointConsumerBuilder resolveEndpointProperties(
```

Comment:
Can this be something like activateBridgeErrorHandler? resolveEndpointProperties confused me on the inboundEndpoint place.